QRCoder/QRCodeGenerator.cs

```diff
@@ -1013,7 +1101,7 @@ private static int GetCountIndicatorLength(int version, EncodingMode encMode)
             }
         }
 
-        private static int GetDataLength(EncodingMode encoding, string plainText, string codedText, bool forceUtf8)
+        private static int GetDataLength(EncodingMode encoding, string plainText, BitArray codedText, bool forceUtf8)
         {
             return forceUtf8 || IsUtf8(encoding, plainText, forceUtf8) ? (codedText.Length / 8) : plainText.Length;
```

Comment:
```suggestion
            return forceUtf8 || IsUtf8(encoding, plainText, forceUtf8) ? (int)((uint)codedText.Length / 8) : plainText.Length;
```
Bit-hack instead of division.